content/en/docs/Reference/catalog-templates.md

```diff
@@ -503,3 +503,340 @@ schema: olm.channel
 
 Here, a channel is generated for each template channel which differs by minor version, and each channel has a `replaces` edge from the predecessor channel to the next-lesser minor bundle version. Please note that at no time do we transgress across major-version boundaries with the channels, to be consistent with [the semver convention](https://semver.org/) for major versions, where the purpose is to make incompatible API changes.
 
+## Composite Template
+A `composite template` is an input schema that accepts a configuration schema for generating a valid FBC for multiple catalogs at a time. This approach may be attractive to catalog maintainers looking to build multiple versions of a catalog while maintaining fewer files and operator authors looking to contribute their operator to multiple catalogs.
+
+### Usage
+```sh
+opm alpha render-template composite [flags] <filename>
+```
+
+| Flag                | Description                                                                            |
+| ------------------- | -------------------------------------------------------------------------------------- |
+| -h, --help          | prints help/usage information                                                          |
+| -o, --output <type> | the output format, can be `yaml` or `json`                                             |
+| -c, --composite-config <file> | File to use as the composite configuration file (default "catalog/config.yaml") |
+| --skip-tls-verify   | skip TLS certificate verification for container image registries while pulling bundles |
```

Comment:
```suggestion
| --skip-tls-verify   | skip TLS certificate verification for container image registries when pulling bundles |
```

Reply:
This flag description comes directly from the help text of the subcommand. You can see the help text here: https://github.com/operator-framework/operator-registry/blob/7835ea9c613ad1009cc36df8ad16e091d7775368/cmd/opm/root/cmd.go#L35

IMO we should keep this the same as the help text

Reply:
/resolve